Client/Frontend/Settings/Clearables.swift

```diff
@@ -164,3 +164,22 @@ class TrackingProtectionClearable: Clearable {
     }
 }
 
+// Clears our downloaded files in the `~/Documents/Downloads` folder.
+class DownloadedFilesClearable: Clearable {
+    var label: String {
+        return NSLocalizedString("Downloaded Files", tableName: "ClearPrivateData", comment: "Settings item for deleting downloaded files")
```

Comment:
Can we move this string into Strings.swift?

Reply:
This is following suit with the rest of the strings in this file :-/